src/output/icons.rs

```diff
@@ -139,11 +148,15 @@ const DIRECTORY_ICONS: Map<&'static str, char> = phf_map! {
     "node_modules"        => Icons::FOLDER_NPM,     // 
     "npm_cache"           => Icons::FOLDER_NPM,     // 
     "pam.d"               => Icons::FOLDER_KEY,     // 󰢬
+    "portage"             => '\u{f30d}',            // 
     "Pictures"            => '\u{f024f}',           // 󰉏
+    "root"                => '\u{f0250}',           // 󰉐
```

Comment:
Why would root be a lock icon?

Reply:
Why not? It is a folder at the root of the filesystem that normal users can not access.